interface/forms/eye_mag/view.php

```diff
@@ -188,10 +188,9 @@
       <meta name="description" content="OpenEMR: Eye Exam" />
       <meta name="author" content="OpenEMR: Ophthalmology" />
 
-      <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','datetime-picker', 'dialog' ,'jscolor' ]); ?>
+      <?php Header::setupHeader(['datetime-picker', 'dialog' ,'jscolor' ]); ?>
```

Comment:
This file still has a lot of CSS in it that is used for Jquery UI that we'll have to remove in the next issue